app/Http/Controllers/Api/Auth/OAuthController.php

```diff
@@ -85,74 +108,37 @@ public function verify(Request $request)
         }
 
         try {
-            $token = $this->proxy([
-                'username' => $request->input('email'),
-                'password' => $request->input('password'),
-                'grantType' => 'password',
-            ]);
+            $token = $this->proxy(['username' => $request->input('email'), 'password' => $request->input('password'), 'grantType' => 'password',]);
```

Comment:
Don't remove indentation
```suggestion
            $token = $this->proxy([
                'username' => $request->input('email'),
                'password' => $request->input('password'),
                'grantType' => 'password',
            ]);
```